compiler/odds.ml

```diff
@@ -34,7 +34,7 @@ let _ =
           let output_file = Sys.argv.(2) and stdlib_file = Sys.argv.(3) in
           let stdlib = Utils.str_of_file stdlib_file in
           let file = open_out output_file
-          in fprintf file "%s\n\n%s\n" stdlib prog; close_out file
+          in fprintf file "%s\n\n%s\n%s\n" stdlib prog "if PLOT:\n\tplt.show()"; close_out file
```

Comment:
Can we move this to another file? And call a function:
Utils.conclude_program file in 
close_out file